test/jdk/javax/swing/JScrollPane/TestMouseWheelScroll.java

```diff
@@ -0,0 +1,121 @@
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+import java.awt.Point;
+import java.awt.Robot;
+import java.awt.event.InputEvent;
+import javax.swing.DefaultListModel;
+import javax.swing.ListModel;
+import javax.swing.JScrollPane;
+import javax.swing.JFrame;
+import javax.swing.JList;
+import javax.swing.SwingUtilities;
+import javax.swing.UIManager;
+import javax.swing.UnsupportedLookAndFeelException;
+
+/*
+ * @test
+ * @key headful
+ * @requires (os.family != "mac")
+ * @bug  6911375
+ * @summary Verifies mouseWheel effect on JList without scrollBar
+ */
+public class TestMouseWheelScroll {
+
+    static JFrame frame;
+    static JScrollPane scrollPane;
+    static Point p;
+    static int width;
+    static int height;
+    static Point viewPosition;
+
+    private static void setLookAndFeel(UIManager.LookAndFeelInfo laf) {
+        try {
+            UIManager.setLookAndFeel(laf.getClassName());
+        } catch (UnsupportedLookAndFeelException ignored) {
+            System.out.println("Unsupported L&F: " + laf.getClassName());
+        } catch (ClassNotFoundException | InstantiationException
+                 | IllegalAccessException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static void main(String[] args) throws Exception {
+
+        for (UIManager.LookAndFeelInfo laf : UIManager.getInstalledLookAndFeels()) {
+            System.out.println("Testing L&F: " + laf.getClassName());
+            SwingUtilities.invokeAndWait(() -> setLookAndFeel(laf));
+            Robot robot = new Robot();
+            robot.setAutoDelay(100);
+
+            try {
+                SwingUtilities.invokeAndWait(() -> {
+                    frame = new JFrame();
+                    JList list = new JList(createListModel());
+                    // disable list bindings
+                    list.getInputMap().getParent().clear();
+                    scrollPane = new JScrollPane(list);
+
+                    scrollPane.setVerticalScrollBarPolicy(JScrollPane.VERTICAL_SCROLLBAR_NEVER);
+                    frame.add(scrollPane);
+                    frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
```

Comment:
```suggestion
                    frame.setDefaultCloseOperation(JFrame.DISPOSE_ON_CLOSE);
```
It's safer for tests.

Reply:
This testcase was failing on macos which needs to be debugged, so it is excluded in mac for now. But as I told in description, the fix is not affecting other regression test in macos.
It's working on other platforms and I have some issue in my local mac system so I will take this debugging of macos once I sort my system.

Regarding your other question, the orientation will be horizontal. Tested against javax/swing/JScrollPane/HorizontalMouseWheelOnShiftPressed/HorizontalMouseWheelOnShiftPressed.java

Reply:
I see, thank you for explanation. Does it make sense to put the test in ProblemList.txt for macOS, so that it's not forgotten the test is to be run on macOS too?

Reply:
We problemlist a test if it fails in a particular platform whereas in this case it is taken care of by @requires so it will not fail.
But no worries, it's in my radar to take it up. I will raise a JBS as soon as this is committed as I need to mention about this test in JBS description.

Reply:
Thanks!